CHANGES.rst

```diff
@@ -8,7 +8,7 @@ Bug Fixes
 '''''''''
 
 * Fix __future__ imports across cells. See `#308
-  <https://github.com/sphinx-gallery/sphinx-gallery/pull/308>` for more details.
+  <https://github.com/sphinx-gallery/sphinx-gallery/pull/308>`_ for more details.
```

Comment:
:disappointed: the syntax for rst link is something I will never get right on the first attempt I am afraid. I have fixed that in master.